include/cetl/cetl.h

```diff
@@ -0,0 +1,45 @@
+/// @file
+/// CETL common header.
+///
+/// Keep this very spare. CETL's desire is to adapt to future C++ standards
+/// and too many CETL-specific definitions makes it difficult for users to switch off of CETL in the
+/// future.
+///
+/// @copyright Copyright (c) 2023 Amazon.com Inc. and its affiliates. All Rights Reserved.
+///
+
+#ifndef CETL_H_INCLUDED
+#define CETL_H_INCLUDED
+
+//
+// @note CETL_ENABLE_DEBUG_ASSERT
+// Define as 1 to enable library assertions. Enabling this in production code is
+// <em>strongly</em> discouraged.
+//
+#ifndef CETL_ENABLE_DEBUG_ASSERT
+#    ifdef DEBUG
```

Comment:
Surely you mean `NDEBUG`?

Reply:
man, this macro section is all wrong. Good catch.